scripts/x-date-pickers-pro.exports.json

```diff
@@ -321,6 +323,7 @@
   { "name": "PickersTextFieldProps", "kind": "TypeAlias" },
   { "name": "PickersTimezone", "kind": "TypeAlias" },
   { "name": "PickersTranslationKeys", "kind": "TypeAlias" },
+  { "name": "PickersYearButton", "kind": "Variable" },
```

Comment:
Should we export `PickersYearButton` if we are not exporting `PickersMonthButton`? 🤔

Reply:
Removed :+1: